frontend/src/pages/settings/account/Account.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
```

Comment:
I like the new format of the page /settings/account, but it feels a bit weird that the "Change password" elements are now shorter than the elements of "Change email address" and "Export all data"

![Capture d’écran de 2021-12-30 09-44-46](https://user-images.githubusercontent.com/47578089/147736631-0c431a15-5327-4a64-901d-4dd66f9eb2e4.png)
"

Reply:
Some inconsistencies were already present on these settings pages before the upgrade.  
They can be fixed separately in #306